index.html

```diff
@@ -172,43 +171,65 @@ <h2>Glossary</h2>
+<p><dfn class="lint-ignore export" data-lt="case-folded">Case folding</dfn> is the process of making two texts which differ only in case identical for comparison purposes, that is, it is meant for the purpose of string matching. This is distinct from [=case mapping=], which is primarily meant for display purposes. As with the default case mappings, Unicode defines default case fold mappings ("case folding") for each Unicode code point. Unicode defines two forms of case folding.</p>
 
-<p><dfn id="def_case_mapping" class="lint-ignore export">Case mapping</dfn> is the process of transforming characters to a specific case, such as UPPER, lower, or Titlecase. For those scripts that have a case distinction, Unicode defines a <em>default</em> UPPER, lower, and Titlecase character mapping for each Unicode code point. Case mapping, at first, appears simple. However there are variations that need to be considered when treating the full range of Unicode in diverse languages.</p>
+<p><dfn data-lt="case-sensitive|Case sensitive matching" class="lint-ignore export">Case sensitive matching</dfn>. Code points are compared directly with no case folding.</p>
+
+<p><dfn data-lt="character encoding|character encoding form" class="lint-ignore export">Character encoding</dfn> or, more formally, a <em>character encoding form</em>. The way a [=coded character set=] is mapped to bytes for manipulation in a computer. Commonly referred to as just the <span class="alt_name">encoding</span>. For examples and further descriptions see <a href="https://www.w3.org/International/articles/definitions-characters/index#charsets">Character encodings: Essential concepts</a>.</p>
+
+<p><dfn class="lint-ignore export">Character set</dfn> or <span class="alt_name">repertoire</span>. The set of characters one might use for a particular purpose – be it those required to support Western European languages in computers, or those a Chinese child will learn at school in the third grade (nothing to do with computers).</p>
 
-<p><dfn id="def_case_folding" class="lint-ignore export">Case folding</dfn> is the process of making two texts which differ only in case identical for comparison purposes, that is, it is meant for the purpose of string matching. This is distinct from <a href="#def_case_mapping" class="termref">case mapping</a>, which is primarily meant for display purposes. As with the default case mappings, Unicode defines default case fold mappings ("case folding") for each Unicode code point. Unicode defines two forms of case folding.</p>
+<p><dfn class="lint-ignore export">Circumgraph</dfn>. A single vowel code point that produces glyphs on more than one side of its consonant base. </p>
```

Comment:
This could probably use an example.

Reply:
Yes, i was thinking the same, but was too busy at the time.

Reply:
Done.